src/components/floatingContainer/floatingContainer.ts

```diff
@@ -0,0 +1,26 @@
+import { LitElement, html } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import FloatingContainerScss from './floatingContainer.scss';
+
+/**
+ * Floating Container.
+ * @slot unnamed - Slot for kd-button options.
+ */
+
+@customElement('kd-float-container')
```

Comment:
Maybe `kd-button-float-container` so the context is clear?

Reply:
Addressed